suggested-companies/es.json

```diff
@@ -0,0 +1,3 @@
+[
+	"cabify",
```

Comment:
Sorry that this isn't linked anywhere prominently but #230 describes what the suggested companies are for. From my understanding, I don't think Cabify fits that description. What do you think?

Reply:
Ah... I just thought it was a list of famous companies in each country, and since we didn't have Spain we could kick it off with the Uber competitor. We can remove that file then, if it doesn't fit.

Reply:
It would indeed be great to have suggested companies for Spain, I agree. But since the wizard on our homepage recommends all users from that country to send requests to those companies, we have to be quite strict on which ones we can include.